data/yam/agama/auto/leap16.sh

```diff
@@ -1,7 +1,7 @@
 set -ex
```

Comment:
we don't use this, we can drop this file.

Reply:
removed